### PR TITLE
Chore: Automate deployments

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: .github/workflows/deploy_mainnet.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Deploy Mainnet
+
+on:
+  push:
+    tags:
+    - v*
+
+env:
+  GRAPHKEY: ${{secrets.GITHUB_TOKEN}}
```
Comment (sohkai) [inline .github/workflows/deploy_mainnet.yml:9]: Should we rename the variable to `GRAPHKEY` or etc?

Comment (facuspagnuolo) [reply .github/workflows/deploy_mainnet.yml:9]: not that fast! is still wip, I'm trying it haha

File: .github/workflows/deploy_testnet.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Deploy Testnet
+
+on:
+  push:
+    branches:
+    - '*'
```
Comment (sohkai) [inline .github/workflows/deploy_testnet.yml:6]: Haha the collisions this produces is going to be interesting. Perhaps we should just target `master`, or an internal development branch?

I was thinking, if we migrate this subgraph into the `aragon-court` repo, we could keep a `graph-staging` branch that pushes to a separate staging subgraph URL.

**Review by promaty — APPROVED** [review]
nice

**Pull request merged**